target/linux/ramips/dts/mt7621_ubnt_edgerouter-x.dtsi

```diff
@@ -101,12 +101,7 @@
 
 		partition@140000 {
 			label = "kernel1";
```

Comment:
If you want to merge these two partitions, why not use a more generic name `kernel`.

Reply:
yes good point